governance/votes/Executive vote - May 10, 2023.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Template - [Executive Vote] Coinbase Custody Onboarding, Risk Parameter Changes, Increase Starknet Bridge Limit, Vesting Stream Management - May 10, 2023
+summary: Onboard Coinbase Custody as RWA014-A, update risk parameters for decentralized collateral types, increase the Starknet Bridge limit to 5 million DAI, vesting stream management for PullUp and Phoenix Labs.
+date: 2023-05-10T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Coinbase Custody Onboarding, Risk Parameter Changes, Increase Starknet Bridge Limit, Vesting Stream Management - May 10, 2023
+
+The Arbitration Facilitator(s) and dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes and additions** will occur within the Maker Protocol:
+- RWA014 (Coinbase Custody) will be onboarded as a new vault type, as detailed below.
+- Multiple risk parameters for decentralized collateral types will be updated, as detailed below.
+- The Starknet Bridge Limit will be increased to 5 million DAI, as detailed below.
+- DAI and MKR streams will be set up for PullUp, as detailed below.
+- The old streams for Phoenix Labs will be `yank`ed, and new streams will be created, as detailed below.
+- The `cap` for MKR_VEST will be increased to 2,200 MKR, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
```

Comment:
```suggestion
This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
```
Would remove the stuff about liquidation as might cause confusion around coinbase custody.